phpstan-baseline.neon

```diff
@@ -0,0 +1,31 @@
+parameters:
+	ignoreErrors:
+		-
+			message: "#^Call to static method create\\(\\) on an unknown class Symfony\\\\Component\\\\HttpClient\\\\HttpClient\\.$#"
+			count: 1
+			path: src/ClientFactory/SymfonyFactory.php
```

Comment:
i guess this means we don't test the factory, otherwise we would depend on the symfony http client.

we should probably add that, but in a separate merge request. and its no blocker for releasing 2.0